server/utils/utils.ts

```diff
@@ -71,3 +71,27 @@ export const extractPrisonerNumber = (search: string): string | false => {
 export const getFormatDistanceToNow = (date: Date) => {
   return formatDistanceStrict(date, new Date())
 }
+
+export const formatDateForApi = (dateOfBirth: string) => {
+  const date = JSON.parse(dateOfBirth)
+  if (date.year && date.month && date.day) {
+    return `${date.year}-${date.month}-${date.day}`
+  }
+  return null
+}
+
+const isListed = (val: string, name: string) => {
+  return val === name || val.toLocaleUpperCase() === name || val.toLocaleLowerCase() === name
```

Comment:
couldn't you convert both to upper/lower case and do it in one condition `val.toLocalUpperCase() === name.toLocalUpperCase()`

Reply:
Code refactored to use one condition (`val.toLocalUpperCase() === name.toLocalUpperCase()`)